core/dbt/include/global_project/macros/adapters/metadata.sql

```diff
@@ -63,3 +63,12 @@
   {{ exceptions.raise_not_implemented(
     'list_relations_without_caching macro not implemented for adapter '+adapter.type()) }}
 {% endmacro %}
+
+{% macro get_relations() %}
+  {{ return(adapter.dispatch('get_relations')) }}
```

Comment:
This is missing the 'dbt' parameter and it's not actually calling the macro, it's just passing it in. I think this line needs to look like this:
```sql
    {{ return(adapter.dispatch('get_relations', 'dbt')()) }}
```